### PR TITLE
LITE-19830 add extra metadata to logging extras

### PR DESCRIPTION
**Pull request by ffaraoneim**
extra metadata:

- account_id
- account_name
- task_id

### CODE REVIEW

**Review by marcserrat — APPROVED** [review]

File: connect/eaas/dataclasses.py [hunk 1]
```diff
@@ -58,18 +58,30 @@ class TaskPayload:
     def to_json(self):
         return dataclasses.asdict(self)
 
+    @classmethod
+    def from_json(cls, data):
```
Comment (marcserrat) [inline connect/eaas/dataclasses.py:62]: as we discussed, better from_dict and to_dict

Comment (marcserrat) [reply connect/eaas/dataclasses.py]: thanks!

File: connect/eaas/dataclasses.py [hunk 2]
```diff
@@ -55,31 +55,48 @@ class TaskPayload:
     correlation_id: str = None
     reply_to: str = None
 
-    def to_json(self):
+    def to_dict(self):
         return dataclasses.asdict(self)
 
+    @classmethod
+    def from_dict(cls, data):
```
Comment (d3rky) [inline connect/eaas/dataclasses.py:62]: Honestly for me dataclasses are something about the type and structure. That's why I don't think it is needed for some methods here. Especially if you need to copy it across all dataclasses. Why not to use existing `dataclasses.asdict()` and implement one like `fromdict(cls, data)`?

File: connect/eaas/manager.py [hunk 3]
```diff
@@ -222,13 +222,13 @@ async def build_bg_response(self, task_data, future):
         """
         Wait for a background task to be completed and than uild the task result message.
         """
-        result_message = TaskPayload(**task_data.to_json())
+        result_message = TaskPayload(**task_data.to_dict())
```
Comment (d3rky) [inline connect/eaas/manager.py:225]: Not sure, but looks like for copying the instances there is a specific method - [link](https://docs.python.org/3/library/dataclasses.html#dataclasses.replace)

File: connect/eaas/worker.py [hunk 4]
```diff
@@ -93,7 +98,7 @@ async def receive(self):
         except TimeoutError:
             pass
 
-    def get_client(self):
+    def get_client(self, task_id):
```
Comment (d3rky) [inline connect/eaas/worker.py:101]: Looks like here you didn't change comment for the function

**Review by d3rky — CHANGES_REQUESTED** [review]

File: connect/eaas/dataclasses.py (hunk 2 above)
Comment (ffaraoneim) [reply connect/eaas/dataclasses.py]: Done!

**Review by marcserrat — APPROVED** [review]

**Review by d3rky — APPROVED** [review]

**Pull request merged**